### ISSUE TITLE
Connection URL encoding

### ISSUE THREAD

**Issue by smokedsalmonbagel** [toluaina/pgsync#65]
PGSync version: 
1.1.24
Postgres version: 

Elasticsearch version: 

Redis version: 

Python version: 
3.7.9
Problem Description: 
If a password or username contains special characters (% in my case) it will not be properly URL encoded for sqlalchemy.  This will result in:
```
FATAL:  password authentication failed for user "someuser"
```
get_postgres_url() in pgsync/utils.py should properly URL encode the string using something like 
```
import urllib.parse
pw = urllib.parse.quote_plus("my%funny&pasword")
```

Error Message (if any): 

```
FATAL:  password authentication failed for user "someuser"
```

**Comment by toluaina** [comment]
Yes, you are absolutely correct. thanks

**Comment by toluaina** [comment]
This is included in the 1.1.25 release

**Comment by smokedsalmonbagel** [comment]
I tested this with 1.1.25 and it works now - so I will close this.  Thanks!
